qemu/tests/rng_bat.py

```diff
@@ -120,17 +118,24 @@ def get_driver_id(session, cmd, pattern):
         logging.info("register 'viorngum.dll' into system")
         session.cmd(rng_dll_register_cmd, timeout=120)
 
+    check_rngd_service = params["check_rngd_service"]
```

Comment:
Should be:
```python
check_rngd_service = params.get("check_rngd_service")
if check_rngd_service:
    ...
```
otherwise it would result in `test.cancel` for windows guests.

Reply:
Updated, thanks!